### PR TITLE
Feature client-login 

### PR DESCRIPTION
**Pull request by rizwans-cuelogic**
1. client-login feature 

pivotal tracker link: https://www.pivotaltracker.com/n/projects/1970105

### CODE REVIEW

File: project4/project/ebs/views.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+					return HttpResponse(json.dumps(response), content_type='application/json')				
+				else:
+					response = {'status':'Error', 'message': "User is inactive try again"}
+			else:
+				response = {'status':'Error', 'message': "Invalid Username and Password"}
+			return HttpResponse(json.dumps(response), content_type='application/json')
+
+def log_out(request):
+	logout(request)
+	f1=UserForm()
```
Comment (deepalib-cuelogic) [inline project4/project/ebs/views.py]: remove unused variables

File: project4/project/ebs/views.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+	else:
+	 	f1=UserForm();
+	 	f2=OrgForm();
+	 	f3=UserLoginForm()
+	 	return render(request,'ebs/home.html',{'f1':f1,'f2':f2,'f3':f3})
+
+def loginresult(request):
+	f1=UserForm()
+	f2=OrgForm()
+	f3=UserLoginForm()
```
Comment (deepalib-cuelogic) [inline project4/project/ebs/views.py]: remove unused variables

**Comment by rizwans-cuelogic** [comment]
forgot-password functionality

File: project4/project/ebs/templates/admin/login.html [hunk 3]
```diff
@@ -31,6 +31,7 @@
 {% endif %}
 
 <div id="content-main">
+asdasdasd
```
Comment (deepalib-cuelogic) [inline project4/project/ebs/templates/admin/login.html]: remove this stuff asdasdasd

File: project4/project/ebs/tests.py [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+	def test_whatever_list_view(self):
+		w = self.create_whatever()
+		url = reverse("login")
+		resp = self.client.get(url)
+
+		self.assertEqual(resp.status_code, 200)
+		self.assertIn(w.title, resp.content)
+"""
+
```
Comment (deepalib-cuelogic) [inline project4/project/ebs/tests.py:62]: Need valid test cases for view functions

**Comment by deepalib-cuelogic** [comment]
Add valid test cases for view functions, resolve review comments.
Please recreate a PR, closing this one.

**Pull request closed without merging**